linkedList.js

```diff
@@ -3,6 +3,85 @@
  * @description  Main Module
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
-function linkedListGenerator(){
+function linkedListGenerator() {
+  let head = null;
+  let tail = null;
+  function getHead() {
+    return head;
+  }
+  function getTail() {
+    return tail;
+  }
+  function node(value) {
+    return { value: value, next: null, previous: null };
+  }
+  function add(value) {
+    let newNode = node(value);
+    if (head === null) {
+      head = newNode;
+    } else {
+      tail.next = newNode;
+      newNode.previous = tail;
+    }
+    tail = newNode;
+    return newNode;
+  }
+  function remove(number) {
+    if (get(number) === false) {
+      return false;
+    }
+    if (number === 0) {
+      head = head.next;
+    }
+    let current = get(number);
+    let previous = get(number - 1);
+    previous.next = current.next;
```

Comment:
This should be placed in an if/else if/else chain along with the if immediately above and the if below. As it stands, when you remove from the head of the list, it'll also remove twice since the if above doesn't stop the function from continuing down to this line.